Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -604,7 +638,11 @@ void FiniteDifferenceSolver::HybridPICSolveECylindrical (
                 // interpolate the nodal neE values to the Yee grid
                 auto enE_r = Interp(enE, nodal, Er_stag, coarsen, i, j, 0, 0);
 
-                Er(i, j, 0) = (enE_r - grad_Pe) / rho_val;
+                if (rho_val < rho_floor && holmstrom_vacuum_region) {
```

Comment:
It's curious to me what the clang-tidy tests focus on. I would think it would prefer this to be
```
if (holmstrom_vacuum_region && (rho_val < rho_floor))
```
due to the possibility to short-circuit and avoid the comparison.